### DISCUSSION TITLE
Fast Forward and backward buttons

### DISCUSSION THREAD

**Discussion by machadoug** [yoriiis/vlitejs#92] [Ideas]
How to add the fast Forward and backward buttons to the player?

**Comment by yoriiis** [comment]
Fast forward and back forward actions are already available with keyboard shortcuts on the player, you can create your own buttons and take inspiration from the [fastForward](https://github.com/vlitejs/vlite/blob/main/src/vlite/assets/scripts/vlite.ts#L313-L317) function.

**Reply by machadoug** [reply]
Thank you for the swift reply. 
I couldn't find in the documentation how to add buttons to the video and audio players.  Can you please give instructions on how to add the buttons?
TIA

**Reply by yoriiis** [reply]
When the player is ready, you can create a button element and inject it into the control bar for example, then attach an event listener to fast forward.

```js
new Vlitejs('#player', {
    onReady: function (player) {
        player.elements.controlBar.insertAdjacentHTML(
            'beforeend',
            '<button class="btn-ff">>></button>'
        );
        document.querySelector('.btn-ff').addEventListener('click', (e) => {
            e.preventDefault();
            player.getCurrentTime().then((seconds) => {
                player.seekTo(seconds + 5);
            });
        });
    }
});
```
